wled00/UISimulator/runSimulator

```diff
@@ -0,0 +1,2 @@
+cd wled00
+sudo python3 UISimulator/main.py "$@" 
```

Comment:
`sudo` should not be needed in a public repository.

Reply:
I don't know what that means.  For example, is it permissible in documentation in a public repository?

Reply:
it means that i don't want to have scripts that use `sudo`. 
Running programs with root permissions is only permissible when there is a very good justification.